packages/pds/src/app-view/api/app/bsky/feed/getPostThread.ts

```diff
@@ -91,6 +91,7 @@ const composeThread = (
   }
 
   if (post.author.viewer?.blocking || post.author.viewer?.blockedBy) {
+    // Globally blocked replies via post.replyBlocked are fully filtered out of results.
```

Comment:
is it possible that we could still return as a `#blockedPost`?

seems like this could lead to unexpected behavior 🤔

Reply:
i almost wonder if it's worth having two different defs for these like `blockedUserPost` & `blockNonComplyingPost`

(names are kinda longwinded but just to get the point across)

Reply:
I really wanna get in that fix to #threadViewPost, we should coordinate with paul on how we might do it